spec/namespaces/notifications.yaml

```diff
@@ -225,6 +236,12 @@ components:
         application/json:
           schema:
             $ref: '../schemas/notifications._common.yaml#/components/schemas/GetConfigsResponse'
+    notifications.list_channels@200:
+      description: ''
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/notifications._common.yaml#/components/schemas/ListChannelsResponse'
```

Comment:
is `ListChannelsResponse` going to be reused reused else where? If not we should just declare the schema here to reduce reference hell. The other endpoints that have this kind of references came from the legacy Smithy->OpenAPI conversion. We will need to update them in a future PR.

Reply:
Good point! done.